CHANGES_NEXT_RELEASE

```diff
@@ -22,3 +22,4 @@
 - [HARDENING] Relocate attr_persistence check from OrionMongoBaseSink to OrionMongoSink (#793)
 - [BUG] Fix recvTime in OrionMongoSink (#796)
 - [BUG] Fix fiware service path field in OrionCKANSink, row mode (#800)
+- [BUG] Fix input value for collection_size and initialization for data_expiration in OrionMongoSink and OrionMongoBaseSink (#803)
```

Comment:
When fixing an issue about a feature done in the same sprint there is no need to update CHANGES_NEXT_RELEASE, since it is considered as part of such a feature.